media-sound/krecorder/krecorder-23.04.49.9999.ebuild

```diff
@@ -0,0 +1,30 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+KFMIN=5.89.0
+QTMIN=5.15.0
+inherit ecm gear.kde.org
+
+DESCRIPTION="A convergent audio recording application for Plasma"
+HOMEPAGE="https://apps.kde.org/krecorder/"
+
+LICENSE="CC0-1.0 CC-BY-4.0 GPL-3+"
+SLOT="0"
+KEYWORDS=""
+
+DEPEND="
+	>=dev-libs/kirigami-addons-0.6:5
+	>=dev-qt/qtdeclarative-${QTMIN}:5
+	>=dev-qt/qtgui-${QTMIN}:5
+	>=dev-qt/qtmultimedia-${QTMIN}:5
+	>=dev-qt/qtquickcontrols2-${QTMIN}:5
+	>=dev-qt/qtwidgets-${QTMIN}:5
+	>=kde-frameworks/kconfig-${KFMIN}:5
+	>=kde-frameworks/kcoreaddons-${KFMIN}:5
+	>=kde-frameworks/ki18n-${KFMIN}:5
+	>=kde-frameworks/kirigami-${KFMIN}:5
```

Comment:
I know CMakeLists.txt claims to require this, but appears to be an `RDEPEND` only. Would be worth trying to move to a RUNTIME check in cmake and put to upstream.

Reply:
`ki18n_install` macro is used in cmake

Reply:
Moved some of them to RDEPEND

Reply:
I was only pointing at kirigami, I probably should have been clearer in text.

Reply:
Kirigami is used in QML

Reply:
yes, but not at build time, was what I was saying ;)

Reply:
Ah, fixed